frontEnd/tickets-app/src/pages/SignIn/signIn.html

```diff
@@ -0,0 +1,12 @@
+<form>
```

Comment:
we don't write html at all, everything is written in `.js` or `.jsx` files. so you can do:
```
const SignIn = () => {
return <form> ... </form>
}

export defaul SignIn;
```
in a `SignIn.js` file